README.md

```diff
@@ -6,8 +6,10 @@
 This example project mainly targets the STM32L496AG-DISCOVERY development kit
 [P-L496G-CELL02](https://www.st.com/en/evaluation-tools/p-l496g-cell02.html)
 along with the optional
-[X-NUCLEO-IKS02A1](https://www.st.com/en/ecosystems/x-nucleo-iks02a1.html)
-sensor board.
+[X-NUCLEO-IKS01A2](https://www.st.com/en/ecosystems/x-nucleo-iks02a1.html#overview)
```

Comment:
I think it would be appropriate to also update the URL for the link accordingly.

Reply:
It's been fixed now.